test/e2e/support/reports.js

```diff
@@ -1,15 +1,15 @@
-const reporter = require('cucumberjs-allure-reporter');
-const fs = require('fs');
-const mkdirp = require('mkdirp');
+// const reporter = require('cucumberjs-allure-reporter');
+// const fs = require('fs');
+// const mkdirp = require('mkdirp');
 
-const xmlReports = `${process.cwd()}/reports/xml`;
+// const xmlReports = `${process.cwd()}/reports/xml`;
 
-// var createXmlDir = (function() {
-//     if (!fs.existsSync(xmlReports)) {
-//         mkdirp.sync(xmlReports);
-//     }
-// })();
+// // var createXmlDir = (function() {
```

Comment:
And these commented-out lines.

Reply:
i have removed the contents of the entire file